config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -177,8 +177,8 @@ periodics:
       - --deployment=node
       - --gcp-project-type=node-e2e-project
       - --gcp-zone=us-west1-b
-      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config.yaml
-      - --node-test-args= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/perf-config.yaml
```

Comment:
it's better to use a separate config file as well, because I have no idea why we would need to run 3 copies of this on different operating systems.